Modules/Server/Sources/PocketCastsServer/Public/ServerEnums.swift

```diff
@@ -66,3 +66,11 @@ public enum PrimaryRowAction: Int32, Codable {
 public enum PrimaryUpNextSwipeAction: Int32, Codable {
     case playNext = 0, playLast = 1
 }
+
+public enum HeadphoneControl: Int32, Codable {
```

Comment:
In the case a new action is added here, does the sync handle that well?

Reply:
I made the conversion between `HeadphoneControl` and the internal `HeadphoneControlAction` exhaustive so it should be clear when a new value is added to update these.

https://github.com/Automattic/pocket-casts-ios/blob/16a548773f58259ae282ea7a0b663d9de30eb888/podcasts/Settings.swift#L1050-L1080

The values do need to be identical between devices which is just defined in our internal documentation. To be clear about the values, I've added the `rawValue` so it's more obvious if we add them in different orders.